style.css

```diff
@@ -87,12 +95,80 @@ li {
   text-align: center;
 }
 
+.portfolio-grid {
+  display: grid;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  gap: 88px;
+}
+
+.portfolio-flex {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  margin: 0 24px;
+}
+
+.portfolio-flex > * {
+  margin: 16px;
+}
+
+.portfolio-heading {
+  font-size: 32px;
+  font-weight: 700;
+  color: #172b4d;
+}
+
+.portfolio-list-first {
+  font-weight: 600;
+  font-size: 13px;
+  color: #344563;
+}
+
+.portfolio-list-dot {
+  height: 8px;
+  width: 8px;
+  background-color: #c1c7d0;
+  border-radius: 50%;
+}
+
+.portfolio-list-secondary {
+  color: #7a869a;
+  font-size: 13px;
+}
+
+.portfolio-text {
+  color: #344563;
+  font-size: 15px;
+}
+
+.language-text {
+  border-radius: 8px;
+  background-color: #ebebff;
+  padding: 4px 12px;
+  color: #6070ff;
+}
+
+button {
+  background-color: #fff;
+  border-color: #396df2;
+  color: #396df2;
+  padding: 12px;
+  font-weight: 500;
+  font-size: 17px;
+  width: 130px;
+  height: 50px;
+  border-radius: 16px;
+  border-width: 1px;
```

Comment:
The border looks a bit thin. You can optionally increase it.

Reply:
> Your portfolio site is coming together nicely ok_hand
> 
> The only changes required are to the borders of the language skills and each portfolio card. They both need a border radius on them.
> 
> ![image](https://user-images.githubusercontent.com/34813339/152642722-0bdb0333-6479-4ae7-b0ee-5c44cb1bb7a9.png)
> 
> ![image](https://user-images.githubusercontent.com/34813339/152642746-5919abab-d102-4e39-97ea-66053f59d5e7.png)

Im already 2 branches ahead, and fixed this in the current branch im on. Do you you think i should fix it on this branch for the experience?

Reply:
the border according to the spec is supposed to be 1px

Reply:
> 

nevermind, i fixed it here

Reply:
It's a good idea to get some practice as you did 👍 . 
Often, when you are working with other team members on the same feature or section of code, you need to rebase newer branches on top of each other.